src/native/mono/monodroid/monodroid-glue.cc

```diff
@@ -504,15 +504,10 @@ MonodroidRuntime::mono_runtime_init ([[maybe_unused]] JNIEnv *env, [[maybe_unuse
 {
 #if defined (DEBUG)
 	RuntimeOptions options{};
-	int64_t cur_time;
```

Comment:
We don't want to remove this check either.  We want _just_ the change to the `--debugger-agent` string.